proposals/0269-New-vehicle-data-ClimateData.md

```diff
@@ -73,7 +73,7 @@ We need to add `ClimateData` for `GetVehicleData`, `SubscribeVehicleData`, `Unsu
 <param name="externalTemperature" type="VehicleDataResult" mandatory="false" deprecated="true" since="X.x">
 	<description>The external temperature in degrees celsius. This parameter is deprecated starting RPC Spec X.x.x, please see climateData.</description>	
 	<history>
-		<param name="externalTemperature" type="VehicleDataResult" mandatory="false" />
+		<param name="externalTemperature" type="VehicleDataResult" mandatory="false" since="1.0" until="X.x" />
```

Comment:
I think this should be since 2.0, per https://github.com/smartdevicelink/sdl_core/blob/master/src/components/interfaces/MOBILE_API.xml#L6025

Reply:
Done